spec/acceptance/activesupport_instrumentation_spec.rb

```diff
@@ -1,6 +1,6 @@
 unless ActiveSupport::Notifications.respond_to?(:subscribed)
   module SubscribedBehavior
-    def subscribed(callback, *args, &block)
+    def subscribed(callback, *args, &_block)
```

Comment:
I think we cant just leave of `&block` entirely, since we are using `yeild`

Reply:
👍